app/helpers/date_helper.rb

```diff
@@ -3,8 +3,8 @@ def format_date_of_birth(date)
     I18n.l(date.to_date, format: :date_of_birth)
   end
 
-  def format_date_of_visit(date)
-    I18n.l(date.to_date, format: :date_of_visit)
+  def format_near_date(date)
```

Comment:
Perhaps it could just be `format_date`, though, as it's used for everything that isn't a date of birth. I'm open to ideas.